src/development/_posts/2016-12-13-algo.md

```diff
@@ -0,0 +1,13 @@
+---
+published: true
+title: A VPN in minutes with Algo
+excerpt: Quickly get up and running with a VPN using your systems native client.
+
+tags: security vpn infrastructure
+---
+
+Over the last few months there’s been a bit of a race to arms with many folk arming themselves with a VPN for protection against Snoopers Charter amongst other reasons. For me personally, it’s as much a means of protecting myself on public WiFi more than anything else, but the additional privacy is certainly a plus.
```

Comment:
1. I'd probably say "Digital Economy Bill" as that's really what it is. Do you have a good reference to what it is, too?

2. Why do you need to protect yourself on public WiFi all of a sudden?